pkgs/tools/audio/jack-tools/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, alsaLib, libGL, libGLU
+,freeglut, libsndfile, libsamplerate, liblo, ncurses, libtool, xorg, asciidoc, libxslt, libxml2 }:
+
+stdenv.mkDerivation rec {
+  name = "jack-tools-${version}";
+  version = "20131226";
+
+  src = fetchgit {
+    url    = https://salsa.debian.org/multimedia-team/jack-tools.git;
+    rev    = "6f2a666d6859e7ef764145a73bdd5985ff850173";
+    sha256 = "003pnyqbhncal5azf8qznz34mja4ywbgbmfkdfyizz12j4awscw4";
+  };
+
+
+#  nativeBuildInputs = [ pkgconfig ];
+ 
+ buildInputs = [
+    libjack2
+    alsaLib
+    libGL
+    libGLU
+    freeglut
+    libsndfile
+    libsamplerate
+    liblo
+    ncurses
+    libtool
+    xorg.libXext
+    xorg.libXt
+    asciidoc
+    libxslt
+    libxml2 
+  ];
+
+#  NIX_CFLAGS_COMPILE = "-fpermissive";
+
+#  configureFlags = [
+#    "--with-faac-prefix=${faac}"
+#    "--with-lame-prefix=${lame.lib}"
+#  ];
+
+  buildPhase = ''
+    make mk-local-c-common
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
Why not rely on the default `installPhase` that runs `make install`?

Reply:
it tries to install to $home/opt :
cp: target '/homeless-shelter/opt/bin' is not a directory